comixed-rest-api/src/main/java/org/comixed/net/ConvertComicsRequest.java

```diff
@@ -20,36 +20,44 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
 import org.comixed.adaptors.ArchiveType;
 
+/**
+ * <code>ConvertComicsRequest</code> provides the HTTP request contents when converting comics.
+ *
+ * @author Darryl L. Pierce
+ */
 public class ConvertComicsRequest {
+  @Getter
+  @Setter
   @JsonProperty("ids")
   private List<Long> comicIdList;
 
+  @Getter
+  @Setter
   @JsonProperty("archiveType")
   private ArchiveType archiveType;
 
+  @Getter
+  @Setter
   @JsonProperty("renamePages")
   private boolean renamePages;
 
-  public ConvertComicsRequest() {}
+  @Getter
+  @Setter
+  @JsonProperty("deletePages")
+  private boolean deletePages;
 
   public ConvertComicsRequest(
-      List<Long> comicIdList, ArchiveType archiveType, boolean renamePages) {
+      final List<Long> comicIdList,
+      final ArchiveType archiveType,
+      final boolean renamePages,
+      final Boolean deletePages) {
     this.comicIdList = comicIdList;
     this.archiveType = archiveType;
     this.renamePages = renamePages;
-  }
-
-  public List<Long> getComicIdList() {
-    return comicIdList;
-  }
-
-  public ArchiveType getArchiveType() {
-    return archiveType;
-  }
-
-  public boolean getRenamePages() {
-    return renamePages;
+    this.deletePages = deletePages;
```

Comment:
For this constructor you can use the annotation @AllArgsConstructor on the class.

Reply:
I don't think that will work since this represents an HTTP request payload: when it gets created in a real-life transaction, I believe Spring only uses the no-arg constructor. So it wouldn't be able to set the fields if they're marked as final, which is how I read Lombok's docs for the AllArgsConstructor annotation.

Is that amistaken understanding on my part?

Reply:
I don't know if this will work... if i put the annotation @AllArgsConstructor and I delombok to see what lombok do i see this : 
  public ConvertComicsRequest(List<Long> comicIdList, ArchiveType archiveType, boolean renamePages)
  {
    this.comicIdList = comicIdList;
    this.archiveType = archiveType;
    this.renamePages = renamePages;
  }

Your own constructor seems to be better with the final keyword... I think we can continue to do without this annotation...